torchvision/transforms.py

```diff
@@ -43,7 +43,10 @@ class ToTensor(object):
     def __call__(self, pic):
         if isinstance(pic, np.ndarray):
             # handle numpy array
-            img = torch.from_numpy(pic.transpose((2, 0, 1)))
+            if len(pic.shape) >= 3:
```

Comment:
You can replace `len(pic.shape)` by `pic.ndim`. Also, if the number of dimensions is > 3, the following line (with the `transpose`) will crash.
Maybe you could do something like
```python
if pic.ndim == 2:
    pic = pic[np.newaxis]
elif pic.ndim == 3:
    pic = pic.transpose((2, 0, 1))
else:
    raise ValueError "only 2D and 3D images accepted, got {}D image".format(pic.ndim)
img = torch.from_numpy(pic)
```